crawl-ref/source/player-stats.cc

```diff
@@ -690,3 +691,132 @@ bool have_stat_zero()
 
     return false;
 }
+
+/*Gnoll ability to shift three points from one attribute to another,
+ *becoming drained if successful.
+ *
+ *Cannot shift below stat zero, and cannot use if drained (checked in
+ *ability.cc)
+*/
```

Comment:
This `*` is supposed to line up with the others.

Reply:
Fixed and rebased into the Shift Attributes commit.